ssl-impl/src/main/java/io/crate/protocols/postgres/ssl/SslConfiguration.java

```diff
@@ -247,8 +245,7 @@ static String checkStorePath(String keystoreFilePath) throws FileNotFoundExcepti
 
 
         PrivateKey exportDecryptedKey()
-                throws KeyStoreException, UnrecoverableKeyException, NoSuchAlgorithmException
-        {
+                throws KeyStoreException, UnrecoverableKeyException, NoSuchAlgorithmException {
```

Comment:
Auto-formatting generally adds noise which makes PRs harder to review. It shadows the actual changes. I generally have turned it off.